reconcile/test/test_vault_utils.py

```diff
@@ -45,3 +45,65 @@ def test_sleep_is_called(sleep):
 
         with pytest.raises(SleepCalled):
             client._auto_refresh_client_auth()
+
+
+def test_key_has_leading_space():
+    # The natural thing to do to gain access to the SecretFormatProblem custom
+    # Exception would be to rehome its import to the top of this test file.
+    # However, the TestVaultUtils class will reimport the vault library and
+    # "undo" the import. Thus, the import will be done in each test method that
+    # uses the exception.
+    from reconcile.utils.vault import SecretFormatProblem
```

Comment:
Please do this import at the top of the file

Reply:
For what purpose? It will have no effect for the reason stated in the comment directly preceeding line 56.